motor_test.py

```diff
@@ -34,7 +34,7 @@ def motor_controller_turn_left(self):
 
     def motor_controller_forward(self):
         
-        stop_time = time.time() + 5
+        stop_time = time.time() + 3
         GPIO.add_event_detect(pin,GPIO.RISING,callback=pulse_handler_right)  
 
         while time.time() <	stop_time:
```

Comment:
```suggestion
        GPIO.add_event_detect(pin,GPIO.RISING,callback=pulse_handler_right)  

        stop_time = time.time() + 3
        while time.time() <	stop_time:
```

Reply:
just so we can be more confident that GPIO command won't change the stop_time condition

Reply:
maybe add description too?